doc/ref/peer.rst

```diff
@@ -38,16 +38,16 @@ only recommended for very secure environments.
       .*:
         - .*
 
-This configuration will allow minions with IDs ending in example.com access
+This configuration will allow minions with IDs ending in ``.example.com`` access
```

Comment:
FYI, I use the [conventional comments](https://github.com/saltstack/salt/pull/63332) method of leaving review comments.

**Suggestion (phrasing, style, non-blocking):** I have a very minor suggestion. We follow the [Google Developer's Style Guide](https://developers.google.com/style/tense) which recommends using the present tense. So the fix for this would be to say "This configuration allows minions..." instead.